sql-plugin/src/main/scala/com/nvidia/spark/rapids/DateUtils.scala

```diff
@@ -21,12 +21,14 @@ import java.time.LocalDate
 import scala.collection.mutable.ListBuffer
 
 import org.apache.spark.sql.catalyst.util.DateTimeUtils
-import org.apache.spark.sql.catalyst.util.DateTimeUtils.{localDateToDays, SQLDate}
+import org.apache.spark.sql.catalyst.util.DateTimeUtils.localDateToDays
 
 /**
  * Class for helper functions for Date
  */
 object DateUtils {
+  type RapidsSQLDate = Int
```

Comment:
Is there any point to this typedef?  It's never used except in this file as a return type.  Spark apparently just reverted to using Int directly, which makes me wonder why we wouldn't follow their lead.

Reply:
I considered this as well, and favored staying close to what we had, but  reasoning for simply switching to Int makes sense to me as well.